embedded/store/immustore.go

```diff
@@ -183,8 +186,9 @@ type ImmuStore struct {
 	blBuffer chan ([sha256.Size]byte)
 	blErr    error
 
-	precommitWHub *watchers.WatchersHub
-	commitWHub    *watchers.WatchersHub
+	precommitWHub        *watchers.WatchersHub
+	durablePrecommitWHub *watchers.WatchersHub
```

Comment:
I'm not convinced also about the name, it was the best I could think of. Didn't wanted to change existent one, but I'd think it could be done even afterwards as well